src/components/password/password.stories.mdx

```diff
@@ -0,0 +1,210 @@
+  <Story name="disabled" story={stories.Disabled} />
+</Canvas>
+
+### ReadOnly
+
+<Canvas>
+  <Story name="readOnly" story={stories.ReadOnly} />
+</Canvas>
+
+### AutoFocus
+
+<Canvas>
+  <Story name="with autoFocus" story={stories.AutoFocus} />
+</Canvas>
+
+### With labelInline
+
+<Canvas>
+  <Story name="with labelInline" story={stories.WithLabelInline} />
+</Canvas>
+
+### With label Align
+
+<Canvas>
+  <Story name="with labelAlign" story={stories.WithLabelAlign} />
+</Canvas>
+
+### With custom labelWidth and inputWidth
+
+<Canvas>
+  <Story name="with custom labelWidth and inputWidth" story={stories.WithCustomLabelWidthAndInputWidth} />
+</Canvas>
+
+### With custom maxWidth
+
+<Canvas>
+  <Story name="with custom maxWidth" story={stories.WithCustomMaxWidth} />
+</Canvas>
+
+### With fieldHelp
+
+<Canvas>
+  <Story name="with fieldHelp" story={stories.WithFieldHelp} />
+</Canvas>
+
+### With labelHelp
+
+<Canvas>
+  <Story name="with labelHelp" story={stories.WithLabelHelp} />
+</Canvas>
+
+### With required
+
+<Canvas>
+  <Story name="required" story={stories.WithRequired} />
+</Canvas>
+
+
+
+### Validations
+
+Validation status can be set by passing `error`, `warning` or `info` prop to the component
+
+Passing a string to these props will display a properly colored border along with a validation icon and tooltip - string value will be displayed as the tooltip message.
+
+Passing a boolean to these props will display only a properly colored border.
+
+For more information check our [Validations](?path=/docs/documentation-validations--page "Validations") documentation page
+
+#### As a string
+
+<Canvas>
+  <Story
+    name="validations - string - component"
+    story={stories.ValidationsAsAString}
+  />
+</Canvas>
+
+It is possible to use the `tooltipPosition` to override the default placement of tooltips rendered as part of this component.
+
+<Canvas>
+  <Story
+    name="validations - string - with tooltipPosition overriden - component"
+    story={stories.ValidationsAsAStringWithTooltipCustom}
+  />
+</Canvas>
+
+#### As a string, displayed on label
+
+<Canvas>
+  <Story
+    name="validations - string - label"
+    story={stories.ValidationsAsAStringDisplayedOnLabel}
+  />
+</Canvas>
+
+#### New designs validation
+
+<Canvas>
+  <Story
+    name="validations - string - new design"
+    story={stories.NewDesignsValidation}
+  />
+</Canvas>
+
+It is possible to use the `tooltipPosition` to override the default placement of tooltips rendered as part of this component.
+
+<Canvas>
+  <Story
+    name="validations - string - with tooltipPosition overriden - label"
+    story={stories.ValidationsAsAStringWithTooltipDefault}
+  />
+</Canvas>
+
+#### As a boolean
+
+<Canvas>
+  <Story name="validations - boolean" story={stories.ValidationsAsABoolean} />
+</Canvas>
+
+## Props
+
+### Password
+
+<StyledSystemProps of={Password} noHeader margin />
+
+**Any other supplied props will be provided to the underlying HTML input element**
+
```

Comment:
Apologies for not spotting this previously @tomdavies73, it would be a good idea to document the new translation keys too (`password.ariaLiveShownMessage` and `password.ariaLiveHiddenMessage`).

We have several examples for other components which should help you with how to structure this